static/js/panel.js

```diff
@@ -76,6 +77,20 @@ var Panel = new (class Panel {
       });
     }
 
+    const updateUnpermalink = () => {
+      if (document.location.pathname.startsWith("/searchfox/rev/")) {
```

Comment:
just realized this is wrong. this works only on test website.
I'll fix this part